dashboard/test/ui/test_status.js

```diff
@@ -125,9 +125,14 @@ Test.prototype.updateView = function() {
   // Update row appearance
   row.className = this.status;
   if (succeeded || failed) {
+    var formattedDuration = new Date(this.duration * 1000)
+      .toISOString() // Sample formatting - 1970-01-01T00:00:42.000Z
+      .slice(11, 19) // extracts the time from the above string
+      .split(":");
+
```

Comment:
I would just create a method called getFormattedDate() that returns the string 
${formattedDuration[0]} hr ${formattedDuration[1]} min ${formattedDuration[2]}

The reason is that it is the only thing this variable is doing and will keep the logic in a single place. We don't really use the rest of the values in the vector for anything else

Reply:
I would also recommend you avoid manually slicing and splitting the ISO string to extract the data you want, and instead use `formattedDuration.getUTCHours()`, `formattedDuration.getUTCMinutes()`, and
 `formattedDuration.getUTCSeconds()`.

Reply:
@pablo-code-org , I'm assuming with the simplification Elijah suggested, we don't have the vector and only a date object. So, leaving the code inline.

Thanks  @Hamms  on the pointer on getUTCHours, this is so much cleaner. I had originally wanted to do this and was using getHours instead, which gave the wrong value.

Reply:
It really is silly that the default/standard methods aren't in UTC by default with something like a `getTimeZoneHours` alternative for the subjective case